### PR TITLE
flesh out media atom to be usable

### PR DESCRIPTION
**Pull request by jamesgorrie**
We've added:
* title - To make sure the video is at least searchable
* category: There was an editorial decision to make these types of videos, so if we are going to add one, it would have to be well thought out
* duration and source: these will be required for publishing, but as we might not have an asset, duration is optional, and source doesn't need to know at first

This will get us a lot closer to getting one of these into prod.

### CODE REVIEW

**Comment by jamesgorrie** [comment]
@akash1810 @paulmr

**Comment by akash1810** [comment]
👍

File: thrift/src/main/thrift/atoms/media.thrift [hunk 1]
```diff
@@ -18,8 +18,8 @@ enum AssetType {
 }
 
 enum Category {
-    EXPLAINER,
     NEWS,
```
Comment (akash1810) [inline thrift/src/main/thrift/atoms/media.thrift]: why not alphabetical ordering?

Comment (jamesgorrie) [inline thrift/src/main/thrift/atoms/media.thrift]: It's not elsewhere, don't suppose it matter - fixed.

**Pull request merged**